netcode.c

```diff
@@ -5328,7 +5341,7 @@ static void test_address()
 
 #define TEST_PROTOCOL_ID            0x1122334455667788ULL
 #define TEST_CLIENT_ID              0x1ULL
-#define TEST_MATCH_ID				0x2ULL
+#define TEST_MATCH_ID			0x2ULL
```

Comment:
It seems the code still doesn't line up, you might want tot use sublime or something, or double-check to make sure it is aligned.